stdlib/Profile/src/Allocs.jl

```diff
@@ -15,13 +15,40 @@ using Base: InterpreterIP
 # The C jl_bt_element_t object contains either an IP pointer (size_t) or a void*.
 const BTElement = Csize_t;
 
-# matches jl_raw_backtrace_t on the C side
+"""
+    RawBacktrace
+
+Represent the raw backtrace information as it is captured at the C level within Julia. 
+It holds a pointer to the raw backtrace data (`data`) and the size of this data (`size`). 
+This struct mirrors the internal C representation used by Julia to manage backtraces, 
+facilitating the transfer of data between Julia and its underlying C implementation 
+for memory allocation profiling.
+
+# Fields
+- `data::Ptr{BTElement}`: A pointer to the beginning of the backtrace data array.
+- `size::Csize_t`: The number of elements in the backtrace data array.
```

Comment:
Not clear that we want to document these internal fields — this is not something a user should touch?

Better to document what can be *done* with this data structure via documented functions, e.g. presumably it is returned by `catch_backtrace()` and can be used with `rethrow` and `show_backtrace`?  (I'm just guessing, should be checked.)

Reply:
If this is an undocumented internal type, then it is pretty much only for people who are reading the source code, in which case a docstring listing the fields isn't very useful.